### DISCUSSION TITLE
0.900.20 - what's new?

### DISCUSSION THREAD

**Discussion by akauppi** [firebase/firebase-js-sdk#4704] [General]
When new releases emerge in `npm`, would it be possible to get a brief changelog-like entry here in discussions? 

This would help us test the new releases better.

**Comment by Feiyang1** [comment]
Good idea! We have been using the Google group (the one you joined when you signed up for the EAP) to send release notes, but not everyone is in that group (though I didn't send an email for 0.900.20 yet). 

We published all compatibility packages in this release and have been testing them internally today. Also we are still working on documentations for them, so we are not ready to ask external developers to try yet. I will send an email to the Google group and create a discussion thread once we are comfortable announcing it more widely. 

To give you a preview, compat packages recreate the v8 API to allow you to upgrade your app without changing any code, then you can upgrade to modular API gradually. For example, if you use both auth and firestore, you can upgrade just auth to the modular API while keeping firestore in the old API style using the compat package. 

You can even mix the API style within the same product. For example, using `firebase.auth.signInWithEmailAndPassword()` from `firebase/compat/auth` and `signOut()` from `firebase/auth` side by side. All of these hopefully will provide an easy upgrade path for developers.

Compat packages are exposed under the submodule `firebase/compat/[product]`, you would import them the same way as today:
```
import firebase from 'firebase/compat/app';
import 'firebase/compat/auth';
...
```

Feel free to try them and file issues if you found any
